navigation/router.js

```diff
@@ -152,6 +153,20 @@ const ShowsNavigator = createStackNavigator(
   }
 );
 
+const LoadingNavigation = createStackNavigator(
```

Comment:
Why stack navigator? You could just pass Loading screen as an argument for Switch Navigator